benefits/eligibility/templates/eligibility/includes/selection-label--senior.html

```diff
@@ -2,17 +2,18 @@
 {% load i18n %}
 
 {% block label %}
-    {% translate "eligibility.pages.index.senior.label" %}
+    {% translate "Older Adult" %}
 {% endblock label %}
 
 {% block description %}
-    {% translate "eligibility.pages.index.senior.description" %}
+    {% translate "You must be 65 years or older. You will need to verify your identity with" %}
 
     <!-- Button to trigger modal -->
+    {# djlint:off #}
     <button id="login" type="button" class="border-0 bg-transparent p-0" data-bs-toggle="modal" data-bs-target="#modal--login-gov">
         <span class="fallback-text color-logo">Login.gov</span>
-    </button>
-    .
+    </button>.
+    {# djlint:on #}
```

Comment:
I thought this `djlint` was a problem for the punctuation but I see that it was already on a separate line. Maybe we can remove this.

Reply:
This will become unnecessary after https://github.com/cal-itp/benefits/pull/1627

Reply:
The `{# djlint:off #}` stuff will all be in the includes.

Reply:
Just rebased and dropped that commit from this PR.